src/main/java/co/omise/Example.java

```diff
@@ -181,7 +186,7 @@ void destroyCustomer() throws IOException, OmiseException, ClientException {
     void listAllDisputes() throws IOException, OmiseException, ClientException {
         Request<ScopedList<Dispute>> request = new Dispute.ListRequestBuilder().build();
         ScopedList<Dispute> disputes = client().sendRequest(request);
-        System.out.printf("no. of disputes: %d", disputes.getTotal());
+        System.out.printf("total no. of disputes: %d", disputes.getTotal());
```

Comment:
Why do we need to print these?

Reply:
Not sure about this @chakrit Do we still need the print out in the example code krub.

Reply:
Oh I just saw that it's in Example. I think it's ok then.

Sorry for another false alarm

Reply:
![](https://media.giphy.com/media/e62i1q8JWxQly/giphy.gif)